scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -366,15 +365,14 @@ task ModelSegments {
     File? gatk4_jar_override
 
     # Runtime parameters
-    Int? mem
+    Int? mem_gb
     String gatk_docker
     Int? preemptible_attempts
     Int disk_space_gb
 
-    # Mem is in units of GB but our command and memory runtime values are in MB
-    Int machine_mem = if defined(mem) then mem * 1000 else 13000
+    Int machine_mem_mb = select_first([mem_gb, 13]) * 1000
     # ModelSegments seems to need at least 3GB of overhead to run
-    Int command_mem = machine_mem - 3000
+    Int command_mem_mb = machine_mem_mb - 3000
```

Comment:
Does this make sense to you?  I mean, shouldn't the difference between command and machine memory just be the OS, hence independent of the GATK command?

Reply:
Not sure, this was from @jsotobroad.  I'd be fine with standardizing the difference everywhere if that works.

Reply:
@jsotobroad can answer this, @davidbenjamin .